src/Runtime/Runtime.php

```diff
@@ -109,40 +110,18 @@ protected function doRun($argv, $output): int
         // Bootstrap: bootstrap site to the level requested by the command (via a 'post-init' hook)
         $status = $application->run($input, $output);
 
-        // Placate the Drush shutdown handler.
+        // Placate the Drush shutdown handler (@todo remove for v13).
```

Comment:
Is this PR targeting Drush 12 or Drush 13?

Reply:
If we remove the shutdown handler in Drush 12, then it might be hard to know whether an external shutdown handler is needed. If this PR targets Drush 13, then maybe the comment should say v14?